test/tools/ossfuzz/protoToYul.cpp

```diff
@@ -1438,7 +1438,7 @@ void ProtoConverter::visit(Statement const& _x)
 
 void ProtoConverter::openBlockScope()
 {
-	m_scopeFuncs.push_back({});
+	m_scopeFuncs.emplace_back(vector<string>{});
```

Comment:
Isn't this equivalent? Wouldn't the "proper" fix be to use `m_scopeFuncs.emplace_back()`?

Reply:
You're right. Merged too soon. Will make a tiny PR for this.